packages/create-pantheon-decoupled-kit/src/templates/next-drupal-app/app/[lang]/layout.tsx

```diff
@@ -0,0 +1,73 @@
+import '@pantheon-systems/nextjs-kit/style.css';
+
+import { Footer } from '@pantheon-systems/nextjs-kit/dist/components/footer';
+import { Header } from '@pantheon-systems/nextjs-kit/dist/components/header';
+import styles from './layout.module.css';
+import {
+	getCurrentLocaleStore,
+	globalDrupalStateStores,
+} from '../../lib/stores';
+
+import './globals.css';
+
+type FooterData = {
+	title: string,
+	url: string,
+	parent: string
+}
+
+async function getData(locale: string) {
+	const store = getCurrentLocaleStore(locale, globalDrupalStateStores);
+	const footerMenu = await store.getObject({
```

Comment:
Seems like there might be a race condition or something going on here: I am occasionally getting the following error when refreshing the page:

```shell
- error app/[lang]/layout.tsx (21:32) @ getObject
- error TypeError: Cannot read properties of undefined (reading 'getObject')
    at getData (./app/[lang]/layout.tsx:24:36)
    at RootLayout (./app/[lang]/layout.tsx:30:30)
    at async Promise.all (index 0)
  19 | 	const store = getCurrentLocaleStore(locale, globalDrupalStateStores);
  20 | 	console.log(store)
> 21 | 	const footerMenu = await store.getObject({
     | 	                              ^
  22 | 		objectName: 'menu_items--main',
  23 | 	});
  24 |
```

Seems to work smoothly when using optional chaining here which shouldn't be necessary to my understanding.

```suggestion
	const footerMenu = await store?.getObject<FooterData[]>({
```